oak_dice/src/evidence.rs

```diff
@@ -74,8 +83,35 @@ pub struct RootLayerEvidence {
 }
 
 impl RootLayerEvidence {
-    pub fn get_tee_platform(&self) -> Option<TeePlatform> {
-        TeePlatform::from_repr(self.tee_platform)
+    pub fn get_tee_platform(&self) -> Result<TeePlatform, String> {
```

Comment:
Could these return values be `anyhow::Result`?

Reply:
Yes, but that would entail adding anyhow as dependency to stage0, which it currently is not. Since stage0 consumes this crate I'm trying to keep deps as limited as possible. 

Thoughts, @conradgrobler @andrisaar?

Reply:
I agree that we don't want anyhow as a dependency in Stage 0. In general I would prefer `Result<T, & 'static str>`. That could also avoid having to call `to_string` on the error strings.

Reply:
Makes sense. Perhaps we should start recording "negative dependencies" somewhere :) i.e. things we'd prefer not to depend on

Reply:
I wonder if something like cargo-deny will allow us to check that as part of CI

Reply:
good point!